### PR TITLE
Added test for Facebook login from About Us page

### PR DESCRIPTION
**Pull request by tarikkaa**

### CODE REVIEW

File: pages/homePage.js [hunk 1]
```diff
@@ -8,6 +8,8 @@ class HomePage extends BasePage {
     get loginButtonHeader() {return new WebButton($('[id="navbarLoginButton"]'), "Login button in account menu")};
     get logoutButtonHeader() {return new WebButton($('#navbarLogoutButton'), "Logout button in account menu")};
     get basketButton() {return new WebButton($('//span[contains(text(), "Your Basket")]'), "Basket button")};
+    get burgerButton() {return new WebButton($('//mat-icon[contains(text(), "menu")]'), "Burger button")};
+    get aboutUsButton() {return new WebButton($('//span[contains(text(), "About Us")]'), "About Us button")};
```
Comment (nbabii) [inline pages/homePage.js]: to change

File: pages/homePage.js [hunk 2]
```diff
@@ -8,6 +8,8 @@ class HomePage extends BasePage {
     get loginButtonHeader() {return new WebButton($('[id="navbarLoginButton"]'), "Login button in account menu")};
     get logoutButtonHeader() {return new WebButton($('#navbarLogoutButton'), "Logout button in account menu")};
     get basketButton() {return new WebButton($('//span[contains(text(), "Your Basket")]'), "Basket button")};
+    get burgerButton() {return new WebButton($('//mat-icon[contains(text(), "menu")]'), "Burger button")};
```
Comment (nbabii) [inline pages/homePage.js]: to cahnge

**Review by nbabii — CHANGES_REQUESTED** [review]

File: pages/homePage.js (hunk 1 above)
Comment (tarikkaa) [reply pages/homePage.js]: Locator changed

File: pages/homePage.js (hunk 2 above)
Comment (tarikkaa) [reply pages/homePage.js]: Locator changed

**Review by nbabii — APPROVED** [review]

**Pull request merged**